dht_bootstrap.go

```diff
@@ -77,7 +77,16 @@ func (dht *IpfsDHT) BootstrapWithConfig(cfg BootstrapConfig) (goprocess.Process,
 		return nil, fmt.Errorf("invalid number of queries: %d", cfg.Queries)
 	}
 
-	proc := periodicproc.Tick(cfg.Period, dht.bootstrapWorker(cfg))
+	proc := dht.Process().Go(func(p goprocess.Process) {
+		for {
+			select {
```

Comment:
This'll need a `<-p.Go(dht.bootstrapWorker(cfg)).Closed()` at the top (to actually fix the bug :smile:). Sorry about that.

Reply:
hahahaha wasn't even thinking about that. yup.